spine-cpp/spine-cpp-lite-codegen/spine-cpp-lite-codegen.py

```diff
@@ -0,0 +1,492 @@
+import re
```

Comment:
This file can go into spine-cpp-lite. No need to add another folder just for a single file.